packages/monaco/src/browser/monaco-outline-contribution.ts

```diff
@@ -49,18 +48,15 @@ export class MonacoOutlineContribution implements FrontendApplicationContributio
 
         this.outlineViewManager.onDidSelect(async node => {
             if (MonacoOutlineSymbolInformationNode.is(node) && node.parent) {
-                let widget = this.editorManager.editors.find(editor => editor.editor.uri.toString() === node.uri);
-                if (!widget) {
-                    widget = await this.editorManager.open(new URI(node.uri));
-                }
-                widget.editor.selection = node.range;
-                widget.editor.revealRange(node.range);
+                this.editorManager.open(new URI(node.uri), {
```

Comment:
`open` will switch focus to the editor, so I can no longer use the cursor to go through the outline tree.

Reply:
It would be nice to add `preserveFocus` option, which by default is `false`.